portfolio/src/main/java/com/google/sps/servlets/FunFactsServlet.java

```diff
@@ -14,27 +13,19 @@ public class FunFactsServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> funFacts = new ArrayList<String>();
-    funFacts.add("Every year, my family and I visit Rosarito, Mexico.");
-    funFacts.add("My favorite movie-theater-snack is Trolli sour gummies.");
-    funFacts.add("To de-stress, I enjoy listening to music and painting.");
-    funFacts.add("I enjoy looking at other people's office setups.");
-    funFacts.add("I am fluent in English and Spanish.");
-    funFacts.add("My best pandemic-purchase was a Nintendo Switch.");
-    funFacts.add("My favorite genres of music are: EDM, Reggaeton, & Indie.");
-    funFacts.add("My first programming language was Java.");
+    String[] funFacts = {
+        "Every year, my family and I visit Rosarito, Mexico.",
+        "My favorite movie-theater-snack is Trolli sour gummies.",
+        "To de-stress, I enjoy listening to music and painting.",
+        "I enjoy looking at other people's office setups.",
+        "I am fluent in English and Spanish.",
+        "My best pandemic-purchase was a Nintendo Switch.",
+        "My favorite genres of music are: EDM, Reggaeton, & Indie.",
+        "My first programming language was Java."};
```

Comment:
Super nitpick: we usually want to add a trailing comma when we have arrays that will span multiple items. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas